awsutil/route53.go

```diff
@@ -168,8 +168,9 @@ func (r *Route53) Delete(in route53.ChangeResourceRecordSetsInput) error {
 func (r *Route53) DescribeResourceRecordSets(zoneID *string, hostname *string) (*route53.ResourceRecordSet, error) {
 	params := &route53.ListResourceRecordSetsInput{
 		HostedZoneId:    zoneID,
-		MaxItems:        aws.String("1"),
+		MaxItems:        aws.String("100"),
 		StartRecordName: hostname,
+		StartRecordType: aws.String("A"),
```

Comment:
For consistency use constants from https://github.com/aws/aws-sdk-go/blob/master/service/route53/api.go#L13538-L13571